.github/workflows/cluster-init.yaml

```diff
@@ -147,5 +147,7 @@ jobs:
             name: cluster-init.${{ needs.build-cluster-init.outputs.cluster-init-version }}.tar
 
         - name: Publish Cluster Init Docker Image
-          run: docker push ${{ steps.build.outputs.cluster-init-image }}
+          run: |
+            docker image load --input cluster-init.${{ needs.build-cluster-init.outputs.cluster-init-version }}.tar
+            docker push cluster-init:${{ needs.build-cluster-init.outputs.cluster-init-version }}
```

Comment:
Im not sure if this is correct or there needs to be `ghcr.io/gojek/turing/` appended in front.

A working previous sample is `Building docker image: ghcr.io/gojek/turing/cluster-init:v0.0.0-build.110-b8b8878` from an [older pipeline](https://github.com/gojek/turing/runs/4135628883?check_suite_focus=true)

Reply:
It indeed must be `ghcr.io/${{ github.repository }}/cluster-init:<version>`. 

The way how it was working before is by setting `DOCKER_REGISTRY` env variable ([here](https://github.com/gojek/turing/blob/865fd5d8c73f9be26072cc34fffec96686a80426/.github/workflows/cluster-init.yaml#L70)). Then, `build-image` [make target takes the value of this variable](https://github.com/gojek/turing/blob/865fd5d8c73f9be26072cc34fffec96686a80426/infra/cluster-init/Makefile#L7) (if it's set) and then the resulting value of `${{ needs.build-cluster-init.outputs.cluster-init-version }}` is fully-quialified name with the registry prefix included.

Here you likely need to do something like:
```yaml
           - name: Publish Cluster Init Docker Image
          env:
            DOCKER_REPOSITORY: ghcr.io/${{ github.repository }}
          run: |
            docker image load --input cluster-init.${{ needs.build-cluster-init.outputs.cluster-init-version }}.tar
            docker tag \
              cluster-init:${{ needs.build-cluster-init.outputs.cluster-init-version }} \
              ${{ env.DOCKER_REPOSITORY }}/cluster-init:${{ needs.build-cluster-init.outputs.cluster-init-version }}
            docker push ${{ env.DOCKER_REPOSITORY }}/cluster-init:${{ needs.build-cluster-init.outputs.cluster-init-version }}
```